samples/highcharts/series-dumbbell/lowmarker/demo.js

```diff
@@ -0,0 +1,204 @@
+}, {
+    name: 'battle-ram',
+    previous: 48,
+    current: 30
+}, {
+    name: 'goblin-giant',
+    previous: 10,
+    current: 30
+}, {
+    name: 'three-musketeers',
+    previous: 18,
+    current: 28
+}, {
+    name: 'goblin-barrel',
+    previous: 57,
+    current: 21
+}, {
+    name: 'ram-rider',
+    previous: 23,
+    current: 20
+}, {
+    name: 'elixir-golem',
+    previous: 4,
+    current: 7
+}, {
+    name: 'skeleton-barrel',
+    previous: 4,
+    current: 3
+}].map(dataPoint => {
+    const isIncrease = dataPoint.previous < dataPoint.current;
+    return {
+        ...dataPoint,
+        low: isIncrease ? dataPoint.previous : dataPoint.current,
+        high: isIncrease ? dataPoint.current : dataPoint.previous,
+        color: isIncrease ?
+            Highcharts.getOptions().colors[2] :
+            Highcharts.getOptions().colors[5]
+    };
+});
+
+// Separate the data into increasing and decreasing series
+const increasingData = [],
+    decreasingData = [];
+data.forEach((dataPoint, index) => {
+    const isIncrease = dataPoint.previous < dataPoint.current,
+        transformedDataPoint = {
+            ...dataPoint,
+            x: index
+        };
+
+    if (isIncrease) {
+        increasingData.push(transformedDataPoint);
+    } else {
+        decreasingData.push(transformedDataPoint);
+    }
+});
+
+// Create the chart
+Highcharts.chart('container', {
+    chart: {
+        type: 'dumbbell',
+        inverted: true
+    },
+    title: {
+        text: 'Change in Card Usage'
+    },
+
+    subtitle: {
+        text: 'Season 21 vs Season 22 <br>Top 1000'
+    },
+
+    tooltip: {
+        pointFormat: 'Season 21: used in deck by <strong>{point.previous}</strong> of 1000 top players<br>' +
+            'Season 22: used in deck by <strong>{point.current}</strong> of 1000 top players'
+    },
+
+    xAxis: {
+        type: 'category',
+        opposite: true
+    },
+
+    yAxis: {
+        title: ''
+    },
+
+    legend: {
+        enabled: false
+    },
+
+    plotOptions: {
+        series: {
+            connectorWidth: 3,
+            marker: {
+                radius: 5,
+                states: {
+                    hover: {
+                        lineWidth: 0
+                    }
+                }
+            },
+            dataLabels: {
+                enabled: true,
+                inside: false,
+                crop: false,
+                overflow: 'allow'
+            }
+        }
+    },
+
+    series: [{
+        name: 'Increase',
+        data: increasingData,
+        marker: {
+            enabled: true,
+            symbol: 'triangleRight'
+        },
+        lowMarker: {
+            enabled: false
+        }
+    },
+    {
+        name: 'Decrease',
+        data: decreasingData,
+        marker: {
+            enabled: false
+        },
+        lowMarker: {
+            enabled: true,
+            symbol: 'triangleLeft',
+            fillColor: Highcharts.getOptions().colors[5]
```

Comment:
Why is this necessary? By default I would expect the low-marker to get fill by the `point.color || series.color` just like the main marker.

Reply:
In dumbbell series there is also the lowColor option which is set by default and takes the priority over the series color. I didn't change it due to backward compatibility. Should I change it?

Reply:
Okay, I didn't know that. Let's keep it.

However, I notice that if I set `lowColor` to `undefined`, the marker takes the color of the series. That is intuitive and makes sense to me. So I think we should do two things with that:
* Update the [API docs](https://api.highcharts.com/highcharts/series.dumbbell.lowColor) with that information.
* Change your demo to use `lowColor: undefined` instead of hard-setting `lowMarker.fillColor`.